src/NzbDrone.Common.Test/Http/HttpClientFixture.cs

```diff
@@ -190,6 +190,38 @@ public void should_throw_on_unsuccessful_status_codes(int statusCode)
             ExceptionVerification.IgnoreWarns();
         }
 
+        [Test]
+        public void should_not_throw_on_suppressed_status_codes()
+        {
+            var request = new HttpRequest($"https://{_httpBinHost}/status/{HttpStatusCode.NotFound}");
+            request.SuppressHttpErrorStatusCodes = new[] { HttpStatusCode.NotFound };
+
+            var exception = Assert.Throws<HttpException>(() => Subject.Get<HttpBinResource>(request));
+
+            ExceptionVerification.IgnoreWarns();
+        }
+
+        [Test]
+        public void should_log_unsuccessful_status_codes()
```

Comment:
Believe this test is already covered  by `should_throw_on_unsuccessful_status_codes`